cinnamon/2.5.x/grafana/4.x/elasticsearch/http-client-metrics.json

```diff
@@ -0,0 +1,606 @@
+{
+  "dashboard": {
```

Comment:
Dashboards in this repo shouldn't have the "dashboard" wrapper so that they can be loaded from the Grafana load json UI. Was loading this tested?